includes/degree-import-functions.php

```diff
@@ -161,6 +161,27 @@ function mainsite_degree_format_post_data( $meta, $program ) {
 		}
 	}
 
+	// We are just getting the first id but we can change this later if needed to catch more quotes
+	$meta['quotes_imported_id'] = (!empty($program->quotes[0])) ? $program->quotes[0] : '';
```

Comment:
```suggestion
	$meta['quotes_imported_id'] = ( ! empty( $program->quotes[0] ) ) ?
	    $program->quotes[0] :
	    '';
```

Some small style-guide updates.